organizations/organizations.go

```diff
@@ -290,3 +290,78 @@ func UpdateLogo(w http.ResponseWriter, r *http.Request) {
 
 	utils.GetSuccess("organization logo updated successfully", nil, w)
 }
+
+
+// transfer workspace ownership
+func TransferOwnership(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	org_collection, member_collection := "organizations", "members"
+
+	org_Id := mux.Vars(r)["id"]
+
+	// Check if organization id is valid
+	orgId, err := primitive.ObjectIDFromHex(org_Id)
+	if err != nil {
+		utils.GetError(errors.New("invalid organization id"), http.StatusBadRequest, w)
+		return
+	}
+
+	// Check if organization id exists in the database
+	orgDoc, _ := utils.GetMongoDbDoc(org_collection, bson.M{"_id": orgId})
+	if orgDoc == nil {
+		utils.GetError(errors.New("organization does not exist"), http.StatusBadRequest, w)
+		return
+	}
+
+	requestData := make(map[string]string)
+	if err := utils.ParseJsonFromRequest(r, &requestData); err != nil {
+		utils.GetError(err, http.StatusUnprocessableEntity, w)
+		return
+	}
+
+	email := requestData["email"]
+	
+	// confirm if email supplied is valid
+	if !utils.IsValidEmail(strings.ToLower(email)) {
+		utils.GetError(errors.New("email is not valid"), http.StatusBadRequest, w)
+		return
+	}
+
+	orgMember, err := FetchMember(bson.M{"email": email})
+
+	if err != nil {
+		utils.GetError(errors.New("user not a member of any work pace"), http.StatusBadRequest, w)
```

Comment:
any workspace or this particular workspace?

Reply:
This is for all workspace

But kind of not needed right? Just an extra search through the database that'll increase processing time

I'll remove it

Reply:
I guess I can just include the org_id parameter from the get go to limit the scope of the search